easybuild/easyconfigs/n/NCCL/NCCL-2.18.3-GCCcore-12.3.0-CUDA-12.1.1.eb

```
@@ -11,7 +11,7 @@ toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
 github_account = 'NVIDIA'
 source_urls = [GITHUB_SOURCE]
 sources = ['v%(version)s-1.tar.gz']
-checksums = ['b4f5d7d9eea2c12e32e7a06fe138b2cfc75969c6d5c473aa6f819a792db2fc96']
+checksums = ['6477d83c9edbb34a0ebce6d751a1b32962bc6415d75d04972b676c6894ceaef9']
```

```suggestion
checksums = [('6477d83c9edbb34a0ebce6d751a1b32962bc6415d75d04972b676c6894ceaef9',
              'b4f5d7d9eea2c12e32e7a06fe138b2cfc75969c6d5c473aa6f819a792db2fc96')]
```

We'll allow both checksums, so that those that have the old download do not get an error.

---

ah, good thinking